properties.cmake

```diff
@@ -39,9 +39,9 @@ set(CLAW_X2T_DRIVER_LIB_DIR "${CMAKE_INSTALL_PREFIX}/libexec/")
 #
 # OMNI Compiler variables
 #
-set(OMNI_GIT_COMMIT_HASH "dd9fbd8a08141e582e83ad8dd5f5267889a23a54" CACHE STRING
+set(OMNI_GIT_COMMIT_HASH "92a35f9dbe3601f6177b099825d318cbc3285945" CACHE STRING
     "OMNI compiler tools GIT commit hash")
-set(OMNI_GIT_REPOSITORY "https://github.com/MeteoSwiss-APN/xcodeml-tools.git" CACHE STRING
+set(OMNI_GIT_REPOSITORY "https://github.com/claw-project/xcodeml-tools.git" CACHE STRING
```

Comment:
Just for my understanding. Why forking for the official OMNI? Are you fixing bug that you don't want to upstream?

Reply:
The current plan is to _try_ pushing upstream, but I don't know what the reaction from the developers of OMNI will be. The majority of the current changes are not relevant to official OMNI compiler, so there is no guarantee that they will be accepted.

Additionally, as far as I understood from official OMNI repo history, it is no longer supported. No active development in years. So it is quite likely that they will not be responsive.

Reply:
we would like to bring all back. This would be a way to decouple time line since we have no control on the review/merging on the xodeml-tolls (we do something similar for cosmo). In principle the for would be in sync

Reply:
Ok fine.

Reply:
It is still under development. Last commit 19 days ago. It's part of the programming environment of Fugaku.